tests/system/actions.go

```diff
@@ -13,13 +14,24 @@ type akResult struct {
 	returnCode int
 }
 
+func splitToArgs(cmdArgs string) []string {
+	re := regexp.MustCompile(`\w+=".*?"|\w+=[\w.:]+|"[^"]+"|[\w-.:/]+`)
```

Comment:
Move `re.MustCompile` to `init`. We don't want this to panic when running on a bad regexp.
It will also compile once.

Also: Document the regexp with what it does. Something like:
```
// `a "b c" d` -> []string{"a", "b c", "d"}
```
Otherwise in 2 weeks we'll forget what it does.

Reply:
done